### PR TITLE
Update wording to be ip-prefix specific.

### PR DESCRIPTION
**Pull request by lawrencewilson**

### CODE REVIEW

File: docs/octopus-cloud/static-ip.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ description: How to find the list of static IP addresses for your Octopus Cloud
 
 The Octopus Cloud is a multi-tenant service with several static IP addresses shared among customers in the same Azure region.
 
-Each Azure region uses a pool of 10 static IP addresses. The static IP for an Octopus Cloud instance will use one from the pool and may change to another static IP from the pool under certain situations.
+Each Azure region uses a pool of static IP addresses. The static IP for an Octopus Cloud instance will use one from the pool and may change to another static IP from the pool under certain situations.
```
Comment (cvandal) [inline docs/octopus-cloud/static-ip.md:9]:
```suggestion
Each Azure region uses a range of static IP addresses. The static IP address for an Octopus Cloud Server will be one from the range and may change to another static IP address within the range under certain situations.
```

File: docs/octopus-cloud/static-ip.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ description: How to find the list of static IP addresses for your Octopus Cloud
...
 
 With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud instance. The built-in worker for your Octopus Cloud instance does not use an IP address from the pool of static IP addresses.
```
Comment (cvandal) [inline docs/octopus-cloud/static-ip.md:11]:
```suggestion
With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud Server. The built-in worker for your Octopus Cloud instance does not use an IP address from the range of static IP addresses.
```

File: docs/octopus-cloud/static-ip.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ description: How to find the list of static IP addresses for your Octopus Cloud
...
 
-A list of static IP addresses your instance could be using can be found within the technical section of the instance details page. 
+A range of IP Addresses that your instance could be using can be found within the technical section of the instance details page.
```
Comment (cvandal) [inline docs/octopus-cloud/static-ip.md:13]:
```suggestion
The range of IP Addresses that your Octopus Cloud Server will use can be found within the technical section of the instance details page. 
```

**Review by cvandal — COMMENTED** [review]
@lawrencewilson @harrisonmeister some suggested changes.

File: docs/octopus-cloud/static-ip.md [hunk 4]
```diff
@@ -6,11 +6,11 @@ description: How to find the list of static IP addresses for your Octopus Cloud
 
 The Octopus Cloud is a multi-tenant service with several static IP addresses shared among customers in the same Azure region.
 
-Each Azure region uses a pool of 10 static IP addresses. The static IP for an Octopus Cloud instance will use one from the pool and may change to another static IP from the pool under certain situations.
+Each Azure region uses a range of static IP addresses. The static IP address for an Octopus Cloud Server will be one from the range and may change to another static IP address within the range under certain situations.
 
-With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud instance. The built-in worker for your Octopus Cloud instance does not use an IP address from the pool of static IP addresses.
+With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud Server. The built-in worker for your Octopus Cloud instance does not use an IP address from the range of static IP addresses.
```
Comment (harrisonmeister) [inline docs/octopus-cloud/static-ip.md:11]:
```suggestion
With a static IP address, you can lock down the ingress and egress communications between a Tentacle in your infrastructure and your Octopus Cloud Server. 

:::hint
**Note:**
The built-in worker for your Octopus Cloud Server does not use an IP address from the range of static IP addresses.
:::
```

**Review by harrisonmeister — APPROVED** [review]
LGTM - made one change to highlight the built-in worker as a hint (I think that addition should highlight that difference better)

**Pull request merged**